IDE/STM32Cube/wolfssl_example.c

```diff
@@ -105,9 +105,10 @@
 #endif /* WOLFSSL_STATIC_MEMORY */
 
 
-/* UART definitions */
+/* This sets which UART to use for the console.  It is something you will have
+ * to configure in STMCubeIDE and then change here. */
 #ifndef HAL_CONSOLE_UART
-#define HAL_CONSOLE_UART huart4
+#define HAL_CONSOLE_UART huart3
```

Comment:
Is it actually required to change the default huart here from 4->3? Ideally if you need 3 it should be setup in the default_conf.ftl.

Reply:
Now that you mention it, I had problems using huart3 switched to 4. Ok, will get rid of this.